src/cluster/backend/zeromq/ZeroMQ-Proxy.cc

```diff
@@ -37,6 +37,7 @@ void thread_fun(ProxyThread::Args* args) {
 
 bool ProxyThread::Start() {
     zmq::socket_t xpub(ctx, zmq::socket_type::xpub);
+    xpub.set(zmq::sockopt::xpub_verbose, 1);
```

Comment:
I am missing context here, but is this left-over debug code  or at least something which we should do conditionally based on `debug_flags` to avoid unneeded overhead?